haskell/src/Monocle/Client.hs

```diff
@@ -39,15 +39,15 @@ data MonocleClient = MonocleClient
   }
 
 -- | Create a HTTP manager
-mkManager :: MonadIO m => m Manager
+mkManager :: IO Manager
```

Comment:
I don't think that is necessary, if this was done for the MonadGraphQL IO instance, then remember that IO implements the MonadIO, so it should work similarly.

```suggestion
mkManager :: MonadIO m => m Manager
```